.github/workflows/tests-php.yml

```diff
@@ -18,7 +18,6 @@ jobs:
           - blocks_editor_integration
           - event_status
           - integration
-          - integrations_plugin_wordpress_seo
```

Comment:
Will this be turned on again in the future?

Reply:
It was forked to another action.